### DISCUSSION TITLE
Gitlab `massageMarkdown` does overly aggressive string replacements

### DISCUSSION THREAD

**Discussion by ghost** [renovatebot/renovate#24063] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.57.4

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I am trying to add `SUPPRESS_DEPLOYMENT` to the `prHeader`. If this is in the commit message (which in turn defaults to the PR description as per our repo configuration) this will end up skipping our CD after merge. 

Sadly the [`massageMarkdown` function on the gitlab platform](https://github.com/renovatebot/renovate/blob/main/lib/modules/platform/gitlab/index.ts#L786) is overly aggressive in trying to make sure to change every last instance of PR to MR, so I end up with `SUPMRESS_DEPLOYMENT` instead, which is not all that useful.

I would propose to change that regex to `\bPR\b` and add a `\bPRs\b` for completeness. 

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Duplicate of #20682, PRs welcome

**Reply by HonkingGoose** [reply, marked as answer]
Issue #20682 was fixed with this PR:

- #24068

The fix is in version `36.57.9`. Please update to that version and tell us if it's still broken.
